gst_plugins/python/savant_rs_video_demux.py

```diff
@@ -222,6 +245,24 @@ def do_state_changed(self, old: Gst.State, new: Gst.State, pending: Gst.State):
             self.is_running = True
             self.expiration_thread.start()
 
+        if old == Gst.State.NULL and new == Gst.State.READY:
+            if self.ingress_module and self.ingress_class_name:
+                self.ingress_pyfunc = init_pyfunc(
+                    self,
+                    self.logger,
+                    self.ingress_module,
+                    self.ingress_class_name,
+                    self.ingress_kwargs,
+                    self.ingress_dev_mode,
+                )
+            else:
+                # for AO RTSP
+                self.logger.debug(
+                    'Ingress filter is not set, '
+                    'using default check for video frame content.'
+                )
+                self.ingress_pyfunc = lambda x: not x.content.is_none()
```

Comment:
Use default filter explicitly (`self.ingress_pyfunc = DefaultInressFilter()`).

Reply:
Used the `DefaultInressFilter` class.